packages/ui/src/assets/svg/icons/chart-bars.svg

```diff
@@ -0,0 +1,6 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

Comment:
should run with `yarn build` or `yarn build:svg` for rebuild computed `icons.svg`